world/templates/geek_box.html

```diff
@@ -1,14 +1,20 @@
 <div id="geek-box-container">
   <div id="geek-box" class="row vert-pad-medium">
-    <div class="small-12 medium-6 columns">
-        <h6>Who made this?</h6>
-        <p>This was based on <a href="http://opb.org/aftershock">Aftershock</a>. Carson MacPherson-Krutsky at the Unversity of Montana had the idea to make it available for other locales and types of disasters. It was adapted by Melinda Minch and Eldan Goldenberg for that purpose.</p>
-        <p>Have questions? Click <a href="mailto:{{ settings.contact_email }}">here</a> to email the maintainers of this specific site.</p>
-        <p>Mailing list signup and social media links go here?</p>
+      <div class="small-12 medium-6 columns">
+        <h6>How does it work?</h6>
+        <p>MissoulaReady is designed to help people prepare for disasters that occur in their area. Disasters don’t strike locations equally so we found it important to give people location
+        specific information in order to properly prepare. MissoulaReady organizes current information and packages it in a way that makes it accessible for any Missoula County
+        resident.</p>
+        <p>This site uses the most up-to-date hazard risk data available for Missoula County, Montana. The descriptions of risk and how to prepare for those risks are based on best information from the American Red Cross and the Federal Emergency Management Agency. Data used for MissoulaReady is available for download here.</p>
```

Comment:
oh yeah! I was wondering where that was going to go. And it's all available right here on teh internet.